Biasing/python/ecal.py

```diff
@@ -52,7 +52,7 @@ def photo_nuclear( detector, generator ) :
 
     # Set run parameters
     if generator.energy == 8.0:
-          xsec_bias = 550.
+          xsec_bias = 1000.
```

Comment:
Let me see what this will look like
```suggestion
          xsec_bias = 5000.
```